_component_design/popovers.md

```diff
@@ -14,81 +14,171 @@ last-modified: 2017-08-17
+- The content area may include text, inputs, or interactive content like a table or visualization.
 
 {% endcolumn %}
-{% column left:"hxCol-8" %}
-{% figure [caption:"Triggered popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered.svg){:width="100%"}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A popover consists of a panel, contextual pointer, heading and buttons."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-default-layout.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Triggered popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered-specs.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+## Pattern Specifications 
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Popover Spacing
+{: .hxSubSectionTitle}
+
+For specs on laying out form fields and body text within a popover, refer to [Inputs and Controls]({{site.baseurl}}/components/inputs-and-controls.html)
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Popover spacing specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-spacing.svg){:width="80%"}
 {% endfigure %}
 {% endcolumn %}
+
 </div>
 
-## Closable Popover
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Popover Specs
+{: .hxSubSectionTitle}
+
+- Popovers **always** have a `box-shadow` applied.
+- Popover **should not** be larger than 500 x 500px. 
+
+{% include note.html content="If more space than the above guide lines is required, look into solving the design challenge with a different Helix pattern."%}
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Popover redline specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-specs.svg){:width="80%"}
+{% endfigure %}
+{% endcolumn %}
+</div>
 
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-Closeable popovers are triggered as a result of...
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Pointer Size and Placement
+{: .hxSubSectionTitle}
+
+The pointer is 16 px wide at the base. The apex is 8 px from the side of the rectangle. The pointer should start 12 px from the corner. The side from which the pointer protrudes depends on the location of the trigger on the screen.
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+- Always have the popover protrude toward the center of the screen.
+- Be sure not to hide important information with the popover.
```

Comment:
Switch `Be sure not to hide` to `Do not hide`.

Reply:
updated.